kn/leden/mongo.py

```diff
@@ -33,9 +38,20 @@ def delete(self):
     def save(self, update_fields=NotImplemented):
         if self._parent is None:
             if '_id' in self._data:
-                self._collection.update({
-                    '_id': self._data['_id']}, self._data)
+                if self._versioned:
+                    self._data['_version'] += 1
+                    result = self._collection.update({
+                        '_id': self._data['_id'],
+                        '_version': self._data['_version']-1}, self._data, w=1)
+                    if result['n'] < 1:
+                        raise MongoWrapperException(
+                            'Could not save - version mismatch?')
```

Comment:
Alhoewel: nu is niet duidelijk of het fout gaat omdat iemand anders een update doet of omdat (bijvoorbeeld) de `_id` ineens niet meer bestaat in de database. Dat onderscheid lijkt me moeilijk te maken, en alleen meer complexiteit te geven voor nauwelijks winst.